src/main/plugins/org.dita.html5/xsl/nav.xsl

```diff
@@ -238,6 +238,9 @@ See the accompanying LICENSE file for applicable license.
                         (not(@format) or @format = 'dita' or @format = 'ditamap') ">
           <xsl:value-of select="@copy-to"/>
         </xsl:when>
+        <xsl:when test="@chunk = 'to-content'">
+                    <xsl:value-of select="substring-before(@href,'#')"/>
```

Comment:
I think this should use the [`dita-ot:resolve-href-path`](https://github.com/dita-ot/dita-ot/blob/325337c9a087da363e881366e5ef58e134ba4c1a/src/main/xsl/common/functions.xsl#L19) function.